changelog.d/4592.bugfix

```diff
@@ -0,0 +1 @@
+Revert requestModelBuild() init block in dev tools controller
```

Comment:
This is correct, but the content here should reflect what will be visible to the user, when applicable.
I suggest:
```suggestion
Fix empty Dev Tools screen issue.
```